src/fdtransfer.h

```diff
@@ -17,6 +17,8 @@
 
 #define RESTARTABLE(call)  ({ ssize_t ret; while ((ret = call) < 0 && errno == EINTR); ret; })
 
+#define MAX_PROBE_LEN  256
```

Comment:
Maybe this could be documented for users?

Reply:
This will be superfluous, IMO. I can't imagine a reasonable function name with more than 256 characters. The only thing we must ensure is that async-profiler does not crash due to out-of-bounds access.